data/csv/isodoseRoi.csv

```diff
@@ -1,11 +1,6 @@
-7310,ISO7310,
```

Comment:
is this a desired modification to isodoseROI.csv?

Reply:
Units are now required in `isodoseRoi.csv`, as users can choose between creating levels in cGy and levels in %. Some isodose levels were removed during testing to speed up the ISO2ROI conversion process. These can be re-added if necessary.